src/Person/Person.jsx

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable react/jsx-filename-extension */
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+const Person = ({ name, age, partnerName }) => {
+  return (
+    <section className="Person">
+      {
+        name && (
+          <h2 className="Person__name">
+            My name is&nbsp;
+            {name}
+          </h2>
+        )
+
+      }
+
+      {
+        age && (
+          <p className="Person__age">
+            I am&nbsp;
+            { age }
+          </p>
+        )
+      }
+
+      {
+        partnerName
+          ? (
+            <p className="Person__partner">
+              My wife&apos;s name is&nbsp;
+              { partnerName }
+            </p>
+          )
+          : <p>I am not married</p>
+      }
+    </section>
+  );
+};
+
+Person.propTypes = {
```

Comment:
I think proptype will be optional as we are going to use typescript soon but if you are using it, you should use it with all properties.